android/app/src/main/java/org/uwaterloo/subletr/pages/listingdetails/ListingDetailsPageUiState.kt

```diff
@@ -0,0 +1,14 @@
+package org.uwaterloo.subletr.pages.listingdetails
+
+import android.graphics.Bitmap
+import org.uwaterloo.subletr.api.models.ListingDetails
+
+sealed interface ListingDetailsPageUiState {
+	object Loading : ListingDetailsPageUiState
+
+	data class Loaded(
+		val listingDetails: ListingDetails?,
```

Comment:
Why is this nullable?
Is there any case where `ListingDetails` will be null when the page is `Loaded`?

Reply:
honestly I was thinking this had to match with stream in the viewmodel having `Optional`
but other than that no reason for why it's nullable, I'll fix it

Reply:
Why does `listingDetailsStream` need to be `Optional`?
The default value that you pass in when you call `subscribeAsState` should be sufficient, no? (especially since the values don't change)